js/src/features/human-resouce/components/HumanResourceCalendar/HumanResourceCalendarUserAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Avatar, Popover } from "antd";
+
+import { EmployeeShort } from "HumanResourceModels";
+
+interface Props {
+  employee: EmployeeShort;
+}
+export const HumanResourceCalendarUserAvatar: React.FC<Props> = ({
+  employee,
+  children,
+}) => {
+  const initials =
```

Comment:
Maybe extract this to function and move to helper? And why `&&`? What if employee has only first name or only last name?

Reply:
Done